src/ui.form-calendar/UCalendarMonthView.vue

```diff
@@ -131,33 +237,23 @@ function getMonthClasses(month) {
   const isMoreThenOneMonthRange =
     props.selectedDateTo && isMoreThanOneMonthDiff(props.selectedDate, props.selectedDateTo);
 
-  if (isCurrentMoth(month) && isNotSelectedDate) {
-    return cx([currentMothAttrs.value.class, isMonthInRange && props.config.inRangeDate]);
-  }
-
-  if (props.range && isSelectedMonth(month) && isMoreThenOneMonthRange) {
-    return cx([props.config.inRangeEdgeDate, props.config.inRangeFirstDate]);
-  }
-
-  if (props.range && isSelectedToMonth(month) && isMoreThenOneMonthRange) {
-    return cx([props.config.inRangeEdgeDate, props.config.inRangeLastDate]);
-  }
-
-  if (props.range && isSelectedMonth(month) && !isMoreThenOneMonthRange) {
-    return cx([props.config.inRangeDate, "rounded-dynamic"]);
-  }
-
-  if (isMonthInRange) {
-    return props.config.inRangeDate;
-  }
-
-  if (isSelectedMonth(month)) {
-    return selectedMonthAttrs.value.class;
-  }
-
-  if (isSameMonth(props.activeMonth, month) && !props.range) {
-    return activeMonthAttrs.value.class;
-  }
+  const isCurrentMonthDate = isCurrentMonth(month) && isNotSelectedDate;
+  const isCurrentMonthInRange = isMonthInRange && isCurrentMonthDate && props.selectedDateTo;
+  const isLastMonthInRange = props.range && isSelectedToMonth(month) && isMoreThenOneMonthRange;
+  const isFirstMonthInRange = props.range && isSelectedMonth(month) && isMoreThenOneMonthRange;
+  const isSingleMonthInRange = props.range && isSelectedMonth(month) && !isMoreThenOneMonthRange;
+  const isActiveMonth = isSameMonth(props.activeMonth, month) && !props.range;
+
+  return {
+    isCurrentMonth: isCurrentMonthDate,
+    isCurrentMonthInRange,
+    isLastMonthInRange,
+    isFirstMonthInRange,
+    isSingleMonthInRange,
+    isMonthInRange,
+    isSelectedMonth: isSelectedMonth(month),
```

Comment:
`isSelectedMonth()` shuld be called in other action related way, like `checkIfSelectedMonth()`